changelog.d/5-internal/fix-hlint-issues-in-wire-api

```diff
@@ -0,0 +1 @@
+Fix HLint issues in libs/wire-api. There are a lot left, but these ones should be safe to not change the application's behavior.
```

Comment:
I would just write "Fix some HLint issues in wire-api". Your phrasing does not seem appropriate for the release notes, but feel free to use that text as commit message for the merge.

Reply:
Yes, that sounds reasonable. :+1:

Reply:
(Fixed according to the comment)